File: app/src/main/java/com/lazyxu/mvvmlazy/test/TestActivity.java
package com.lazyxu.mvvmlazy.test;

import android.graphics.Bitmap;
import android.os.Bundle;
import android.os.PersistableBundle;
import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.bumptech.glide.Glide;
import com.bumptech.glide.request.target.SimpleTarget;
import com.bumptech.glide.request.transition.Transition;

public class TestActivity  extends AppCompatActivity {
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        TextView textView=new TextView(this);
        textView.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

            }
        });

    }
    private View.OnClickListener clickListener=new View.OnClickListener() {
        @Override
        public void onClick(View v) {

        }
    };
}

File: app/src/main/java/com/lazyxu/mvvmlazy/test/TestActivity.kt
package com.lazyxu.mvvmlazy.test

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity
import com.alibaba.android.arouter.facade.annotation.Route
import com.lazyxu.base.base.BaseActivity
import com.lazyxu.base.base.head.HeadBuild
import com.lazyxu.base.base.head.HeadToolbar
import com.lazyxu.base.router.RouterUrl
import com.lazyxu.base.utils.SystemUtil
import com.lazyxu.mvvmlazy.R
@Route(path = RouterUrl.App.TEST)
class TestActivity : AppCompatActivity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_test)
    }



    override fun onResume() {
        super.onResume()
        Log.d("HandlerTag", "${SystemUtil.isForeground(this, "com.lazyxu.mvvmlazy.test.TestActivity")}")

    }
}